en/hosting_guide/site_creation.rst

```diff
@@ -0,0 +1,104 @@
+steps to enhance the site features. Those steps are the following:
+
+* Declaring the website in Open Orchestra
+* Dealing with access rights
+* Contributing the home page
+* Configuring the webserver
+* (Optionnal) Configure the 404 & 503 pages
+* (Optional) Generate the robots.txt
+* (Optional) Generate the sitemap.xml
+
+Declaring the site
+------------------
+The first thing to do is to declare the site in Open Orchestra by filling the website creation form.
+
+Full details are available in the user guide, on `website creation page`_.
+
+Dealing with access rights
+--------------------------
+To be able to contribute to the site, access rights have to be set. To do this, create a new group
+including several roles related to this site (including ROLE_ACCESS_TREE_NODE) and assign it to the
+newly created site.
+
+Once this is done, you can add users to the group by editing them. On their next connection, these
+users will be able to select that site with the site switcher. 
+
+.. image:: ../../images/site_switcher.png
+
+Further information on group contribution is available in the user guide, on `user page`_
+
+Contributing the home page
+--------------------------
+To get sure the site is accessible, you should have something to see. So first thing is to select
+the newly created site on the site switcher. Then, under the Editorial part of the menu, the
+'Pages' entry should look like this:
+
+.. image:: ../../images/new_site_tree.png
+
+Clicking on 'New page' will allow you to contribute the homepage of the site. You can go to the
+user guide, on the `node page`_ and `node configuration page`_ for full details about how to edit a
+node.
+
+Once contributed, publish the node (Ideally, in all the languages used by your site aliases).
+
+Configuring the webserver
+-------------------------
+Now, Open Orchestra knows about your site and a first page is contributed. The last thing to do is
+to get that page reachable. To do this you have to configure your webserver. The type of
+configuration depends on you server type, for instance on Apache this is done via the Virtual Host
+mechanism. Here is a simple example in a such context:
+
+::
+
+    <VirtualHost *:80>
+        ServerAdmin webmaster@mysite.com
+        ServerName mysite.com
+
+        DocumentRoot /absolute/path/to/Open/Orchestra/web
+        <Directory /absolute/path/to/Open/Orchestra/web>
+            Options -Indexes +FollowSymLinks -MultiViews
+            AllowOverride All
+            Order allow,deny
+            allow from all
+        </Directory>
+
+        ErrorLog /absolute/path/to/logs/mysite.openorchestra.error.log
+        CustomLog /absolute/path/to/logs/mysite.openorchestra.access.log combined
+    </VirtualHost>
+
+
+Note that the DocumentRoot must point to the web directory of Open Orchestra installation, not on
+the Open Orchestra root directory.
+
+Once Apache restarted, your new website should be accessible.
```

Comment:
You don't need to restart apache, a simple reload will reload the configuration (if I have a good memory)